jedi_vim.py

```diff
@@ -254,7 +254,7 @@ def goto(mode="goto", no_output=False):
                         tagname = d.name
                         while vim_eval('taglist("^%s$")' % tagname) != []:
                             tagname = d.name + ' ' + ''.join(
-                                [random.choice(string.lowercase)
+                                [random.choice(string.lower)
```

Comment:
Ok. I just realized `string` is a module. `string.lowercase` doesn't exist in Python 3 anymore. It's named `ascii_lowercase` there.

You can just defined it manually:

```
>>> string.lowercase
'abcdefghijklmnopqrstuvwxyz'
```